package android_sensorfirewall;

option optimize_for = LITE_RUNTIME;

message SensorEntry {
	required int64 count = 1;
}

message AppEntry {
	required uint32 uid = 1;
	required string pkgName = 2;
	required uint64 lastUpdate = 3;
	repeated SensorEntry sensorEntry = 4;
}

message SensorCounter {
	repeated AppEntry appEntry = 1;
}
